options.go

```diff
@@ -49,3 +54,25 @@ func WithHystrix() clientOption {
 		},
 	}
 }
+
+const (
+	doNotNotifyKey = "interceptor-do-not-notify"
+)
+
+// IgnoreErrorNotification defines if Coldbrew should send this error to notifier or not
+// this should be called from within the service code
+func IgnoreErrorNotification(ctx context.Context, ignore bool) context.Context {
```

Comment:
Expected errors should not be notified while runtime errors should be. 
for expected errors, should I invoke by setting the flag and then unset the flag again ?

Reply:
when you trigger an expected error, just make sure you set the flag as well

Reply:
actually I think coldbrew dont need to notify unless it’s panic? Service owner should have 100% control over what they want to send to sentry, when an error is returned it would be either expected error or already called notifier manually?